### PR TITLE
Switch SwiftFormat to lint mode and have it be lenient on local build…

### PR DESCRIPTION
**Pull request by stefanceriu**
…s but fail on the CI

### CODE REVIEW

File: ElementX.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1493,7 +1486,6 @@
 				E74CD7681375AD2EAA34D66B /* Authentication */,
 				4009BE2E791C16AC6EE39A7E /* BugReport */,
 				B53CA9BECD3F97805E1432D0 /* HomeScreen */,
-				3180C73BA7B8F5F7447C99B0 /* React */,
```
Comment (pixlwave) [inline ElementX.xcodeproj/project.pbxproj:1496]: What was this?

**Review by pixlwave — APPROVED** [review]
LGTM. Presumably without the warning of the missing tool, the build will hard fail? I wonder if we should do that with the other 2 tools too 🤔

File: ElementX.xcodeproj/project.pbxproj (hunk 1 above)
Comment (stefanceriu) [reply ElementX.xcodeproj/project.pbxproj:1496]: Absolutely no idea, it seems you introduced it in https://github.com/vector-im/element-x-ios/blob/26d8affaa8821971a21e159467de19fca337b635/ElementX.xcodeproj/project.pbxproj#L845

Comment (pixlwave) [reply ElementX.xcodeproj/project.pbxproj:1496]: Oh weird, I have no idea what thats about. Good riddance lol!

**Pull request merged**